### PR TITLE
Clean up README and add example for `rustup man`

### PR DESCRIPTION
**Pull request by tesuji**
[Rendered version](https://github.com/lzutao/rustup.rs/blob/readme/README.md).

r? @kinnison

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -281,7 +281,7 @@ $ rustup install stable-msvc
 Toolchain names that don't name a channel instead can be used to name
 [custom toolchains].
 
-[MSVC-based toolchain]: https://www.rust-lang.org/downloads.html#win-foot
+[MSVC-based toolchain]: https://www.rust-lang.org/tools/install
```
Comment (dwijnand) [inline README.md]: Should we link this to https://forge.rust-lang.org/other-installation-methods.html#stable ?

File: README.md [hunk 2]
```diff
@@ -456,7 +456,7 @@ $ rustup target add i686-pc-windows-msvc
 $ rustup target add i686-pc-windows-gnu
 ```
 
-[d]: https://www.rust-lang.org/downloads.html#win-foot
+[d]: https://www.rust-lang.org/tools/install
```
Comment (dwijnand) [inline README.md]: Same here. https://forge.rust-lang.org/other-installation-methods.html#stable ?

**Review by dwijnand — COMMENTED** [review]
LGTM.  1 question (asked twice).

File: README.md (hunk 1 above)
Comment (tesuji) [reply README.md]: I don't have any suggestions. I just open rust-lang/www.rust-lang.org#726 to request a mean to view Windows installation page without platform auto-detection.

Comment (kinnison) [reply README.md]: If https://github.com/rust-lang/www.rust-lang.org/pull/729 gets merged then https://www.rust-lang.org/tools/install?platform_override=win ought to work.

File: README.md [hunk 3]
```diff
@@ -258,30 +258,30 @@ Standard release channel toolchain names have the following form:
 <host>          = <target-triple>
 ```
 
-'channel' is either a named release channel or an explicit version
-number, such as "1.8.0". Channel names can be optionally appended with
-an archive date, as in 'nightly-2014-12-18', in which case the
+`channel` is either a named release channel or an explicit version
+number, such as `1.8.0`. Channel names can be optionally appended with
+an archive date, as in `nightly-2014-12-18`, in which case the
```
Comment (nrc) [inline README.md]: I think this change is incorrect - the quoted text are quotes, not code.

**Review by nrc — COMMENTED** [review]
Thanks for the PR, looks good. I guess we're waiting on the links issue to be resolved before merging this.

**Comment by tesuji** [comment]
rust-lang/www.rust-lang.org#729 gets merged, now https://www.rust-lang.org/tools/install?platform_override=win works.

File: README.md (hunk 3 above)
Comment (tesuji) [reply README.md]: I found it's easier to read by using backtick. If you have strong objection with it, I will change it back.

**Review by dwijnand — APPROVED** [review]

**Pull request merged**